libsrc/target/tvc/asm_tvc_vmode.asm

```diff
@@ -0,0 +1,25 @@
+;   Videoton TV Computer C stub
+;   Sandor Vass - 2019
+;
+;	Set video mode
+;
+
+	SECTION code_clib
+	PUBLIC  asm_tvc_vmode
+    INCLUDE "target/tvc/def/tvc.def"
+
+;
+; Entry:        0, 1, 2 - 2c, 4c, 16c mode
+;
+.asm_tvc_vmode
```

Comment:
Regarding my comment above - it looks like this asm_ function has got C parameter parsing. In this case, it would be best named tvc_vmode and line 21 gains the asm_tvc_vmode label.

Reply:
asm_ files refactored.